templates/repo/commits_list.tmpl

```diff
@@ -16,7 +16,7 @@
 					<td class="author">
 						<div class="tw-flex">
 							{{$userName := .Author.Name}}
-							{{if .User}}
+							{{if and .User (neq .User.ID 0)}} /* User with id == 0 is a fake user from git author */
```

Comment:
Does the actions user (`id == -1`) have an avatar?

Reply:
de4bb2b